third-party/realdds/include/realdds/dds-time.h

```diff
@@ -0,0 +1,127 @@
+// License: Apache 2.0. See LICENSE file in root directory.
+// Copyright(c) 2022 Intel Corporation. All Rights Reserved.
+
+#pragma once
+
+#include <realdds/dds-defines.h>
+#include <fastdds/rtps/common/Time_t.h>
+
+#include <string>
+
+
+namespace realdds {
+
+    
+inline dds_time now()
+{
+    dds_time t;
+    dds_time::now( t );
+    return t;
+}
+
+
+inline dds_time time_from( dds_nsec nanoseconds )
+{
+    dds_time t;
+    t.from_ns( nanoseconds );
+    return t;
+}
+
+
+// Easy way to format DDS time to a legible string, in milliseconds
+//
+class ms_s
```

Comment:
The name is unclear. If you must keep it short than `ms_str` seems better.

Reply:
What if it was just `msec`?
`os << "time offset= " << msec( 5, msec::rel ) << std::endl;`
Or `timestr`?
`os << "time offset= " << timestr( 5, timestr::rel ) << std::endl;`

Reply:
I kind of like `timestr` which makes it a little less specific and fits in with `dds_time`... Objections?

Reply:
`timestr` sounds good to me